examples/extended/src/integration-test/groovy/test/ProfileServiceSpec.groovy

```diff
@@ -0,0 +1,73 @@
+package test
+
+import grails.test.mixin.integration.Integration
+import grails.gorm.transactions.Rollback
+import spock.lang.Specification
+import org.hibernate.SessionFactory
+
+@Integration
+@Rollback
+class ProfileServiceSpec extends Specification {
+
+    ProfileService profileService
+    SessionFactory sessionFactory
+
+    private Long setupData() {
+        // TODO: Populate valid domain instances and return a valid ID
```

Comment:
Can all of this commented out code be deleted?

Reply:
fixed in next commit